core/src/main/java/org/elasticsearch/env/NodeEnvironment.java

```diff
@@ -637,8 +613,8 @@ public boolean hasNodeFile() {
      * Returns all shard paths excluding custom shard path. Note: Shards are only allocated on one of the
      * returned paths. The returned array may contain paths to non-existing directories.
      *
-     * @see #hasCustomDataPath(org.elasticsearch.common.settings.Settings)
-     * @see #resolveCustomLocation(org.elasticsearch.common.settings.Settings, org.elasticsearch.index.shard.ShardId)
+     * @see IndexSettings#hasCustomDataPath()
+     * @see #resolveCustomLocation(IndexSettings, org.elasticsearch.index.shard.ShardId)
```

Comment:
Remove full path from ShardId (it's imported)
`* @see #resolveCustomLocation(IndexSettings, ShardId)`